conda.recipe/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: dask
+  # GIT_DESCRIBE_TAG is supposed to return the latest version (0.4.0) but it is
+  # not working for some reason. And PKG_VERSION is incorrect (it says 1.0.1k)
+  version: {{ environ.get('GIT_DESCRIBE_TAG', '0.4.0')}}
```

Comment:
I think this only works when the build is triggered by a git hook. Not when I trigger it manually. Right @malev ?